aws/secrets.mk

```diff
@@ -17,6 +18,14 @@ CMD_SERVICE_SECRETS_DELETE = @ (echo $(foreach item, $(shell aws ssm --profile=$
 		$(shell aws ssm --profile=$(AWS_PROFILE) delete-parameter --name $(item))) > /dev/null ) && \
 		echo "\033[32m[OK]\033[0m /$(ENV)/$(SVC)/* secrets deleted" || echo "\033[31m[ERROR]\033[0m /$(ENV)/$(SVC)/* secrets deletion"
 
+CMD_SERVICE_ALL_SECRET_KEYS = $(foreach item, $(shell aws ssm --profile=$(AWS_PROFILE) get-parameters-by-path \
+		--path "/$(ENV)/$(SVC)" --recursive | $(JQ) -e -r '.Parameters[] | select(.Type == "SecureString") | .Name' ), $(item))
+CMD_SERVICE_SECRETS_PULL = @ (echo $(shell echo "{\"INFO\":\"EMPTY_JSON_CREATED\"}" > $(SERVICE_SECRETS_BACKUP_FILE)) && \
+		$(foreach item, $(CMD_SERVICE_ALL_SECRET_KEYS), \
+		$(shell $(JQ) --arg value "$(shell echo $(item) | sed 's|.*/||')" '.[$$value] = "'$(shell aws ssm --profile=$(AWS_PROFILE) get-parameter --with-decryption --name $(item) --query Parameter.Value)'"' \
+		$(SERVICE_SECRETS_BACKUP_FILE) > tmp.json && mv tmp.json $(SERVICE_SECRETS_BACKUP_FILE))) > /dev/null ) && \
+		$(JQ) 'del(."INFO")' $(SERVICE_SECRETS_BACKUP_FILE) > tmp.json && mv tmp.json $(SERVICE_SECRETS_BACKUP_FILE) && \
+		echo "\033[32m[OK]\033[0m /$(ENV)/$(SVC)/* secrets pulled" || echo "\033[31m[ERROR]\033[0m /$(ENV)/$(SVC)/* secrets getting"
```

Comment:
When pull fails I'm still getting an OK, so maybe we should check for an exit code somewhere.
(why it failed is unrelated, something local to my environment)
```
/configprovider.py", line 459, in provide
    scoped_config = self._session.get_scoped_config()
  File "/Users/dmitry/.pyenv/versions/3.8.2/lib/python3.8/site-packages/botocore/session.py", line 340, in get_scoped_config
    raise ProfileNotFound(profile=profile_name)
botocore.exceptions.ProfileNotFound: The config profile (nutcorp-) could not be found

[OK] /dev/api/* secrets pulled
```

Reply:
@AutomationD I've checked, yes, this verification doesn't work as expected and it looks like it doesn't work for other targets as well because I just copied this commands. Thanks for finding - will take a look